service/src/main/resources/application.yml

```diff
@@ -69,7 +69,7 @@ wds:
 
 leonardo:
   baseUri: "https://leonardo.dsde-dev.broadinstitute.org/"
-  wdsAppTypeNames: ['WDS', 'CROMWELL']
+  appTypeNames: ['WDS', 'CROMWELL']
```

Comment:
So this was originally an order of preference for "which app do we look for to represent WDS", instead of just "what apps do we know about". We probably will want something similar for CROMWELL when we migrate from CROMWELL to (WORKFLOWS + CROMWELL_RUNNER). If someone is in the "old" world they will want CROMWELL, and new world will want CROMWELL_RUNNER.

Reply:
So in the "new" world, these appTypes will be `CROMWELL_RUNNER, WORKFLOWS, CROMWELL` only?

Reply:
Does this require an update too to cromwhelm and terra-helmfile charts?